modules/command/dv/dv-purchase-key-deposited-command.js

```diff
@@ -159,25 +159,23 @@ class DvPurchaseKeyDepositedCommand extends Command {
         return command;
     }
 
-    async _updatePrivateDataInDb(dataSetId, otObjectId, privateData) {
+    async _updatePermissionedDataInDb(dataSetId, otObjectId, permissionedData) {
         const otObject = await this.graphStorage.findDocumentsByImportIdAndOtObjectId(
             dataSetId,
             otObjectId,
         );
         const documentsToBeUpdated = [];
-        const calculatedPrivateHash = ImportUtilities
-            .calculatePrivateDataHash({ data: privateData });
+        const calculatedPermissionedDataHash = ImportUtilities
+            .calculatePermissionedDataHash({ data: permissionedData });
         otObject.relatedObjects.forEach((relatedObject) => {
             if (relatedObject.vertex.vertexType === 'Data') {
                 const vertexData = relatedObject.vertex.data;
-                constants.PRIVATE_DATA_OBJECT_NAMES.forEach((private_data_array) => {
-                    const privateObject = vertexData[private_data_array];
-                    if (privateObject && privateObject.isPrivate
-                    && privateObject.private_data_hash === calculatedPrivateHash) {
-                        privateObject.data = privateData;
-                        documentsToBeUpdated.push(relatedObject.vertex);
-                    }
-                });
+                const permissionedObject = vertexData.permissioned_data;
+                if (permissionedObject &&
+                    permissionedObject.permissioned_data_hash === calculatedPermissionedDataHash) {
```

Comment:
We should create method that is called validatePermissionedDataHash and use it on all places that we have root hash check